ceph/ceph.py

```diff
@@ -1942,7 +1942,7 @@ def exec_command(self, cmd, **kw):
             self.node.exec_command(
                 cmd=" ".join([self.container_prefix, cmd.replace("sudo", "")]), **kw
             )
-            if self.containerized
+            if self.containerized and not build.startswith("5")
```

Comment:
In case of 5.x, this could be a problem instead opt to use `if not build.startswith("5") and self.containerized`

Reply:
@psathyan Sorry, I'm not able to speculate the problem, would be a great help if you can give me a hint

Reply:
@VasishtaShastry The reason for the change is to avoid a situation wherein the `self.containerized` variable is not initialized.    As the change is common across all versions of the product, we know for sure that `self.containerized` is not initialized when the environment is 5.0 hence it is better to check it first instead of the code raising an exception by evaluating `self.containerized`.

Hope this clarifies the query